packages/amplication-server/src/core/environment/environment.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from 'nestjs-prisma';
+
+import {
+  Environment,
+  CreateEnvironmentArgs,
+  UpdateOneEnvironmentArgs
+} from './dto';
+import { FindOneArgs } from 'src/dto';
+
+@Injectable()
+export class EnvironmentService {
+  constructor(private readonly prisma: PrismaService) {}
+
+  async createEnvironment(args: CreateEnvironmentArgs): Promise<Environment> {
```

Comment:
Why is this method not exposed by the resolver?

Reply:
Currently, each application only have a single environment which is created by default with the application.
At this stage, the user cannot add more environments.